src/services/settings.actions.ts

```diff
@@ -23,16 +23,23 @@ type Opts = typeof SETTINGS_OPTIONS
 export async function loadSettings(): Promise<void> {
   Logs.info('Settings.loadSettings()')
 
-  const stored = await browser.storage.local.get<Stored>('settings')
-
+  // The catch reason: https://bugzilla.mozilla.org/show_bug.cgi?id=1868153
+  const stored = (await browser.storage.managed.get<Stored>('settings').catch(() => {})) ?? {}
```

Comment:
Might want to rename for clarity. With current naming, it took me an extra moment to work through the `normalizeObject` logic below
```suggestion
  const storedManaged = (await browser.storage.managed.get<Stored>('settings').catch(() => {})) ?? {}
```